src/main/java/core/basesyntax/db/DataBase.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataBase {
+    public static final Map<String, Integer> FRUIT_DATABASE = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<String, Integer> database = new HashMap<>();
```
it's not deeply immutable (you make changes to hashmap), therefore it's not a constant